cli/server.go

```diff
@@ -2722,6 +2722,12 @@ func parseExternalAuthProvidersFromEnv(prefix string, environ []string) ([]coder
 			provider.DisplayName = v.Value
 		case "DISPLAY_ICON":
 			provider.DisplayIcon = v.Value
+		case "MCP_URL":
+			provider.MCPURL = v.Value
+		case "MCP_TOOL_ALLOWLIST":
```

Comment:
Since these are regexes and not lists maybe they should be `MCP_TOOL_ALLOW_REGEX` instead